src/Client.ts

```diff
@@ -34,14 +35,14 @@ export class Client implements IHasHTTP {
    * @param timeout - how long to wait before giving up when making ReST calls
    */
   public static async checkServer(server: OnmsServer, httpImpl?: IOnmsHTTP, timeout?: number): Promise<boolean> {
-    const opts = new OnmsHTTPOptions(timeout, server.auth, server);
+    let opts = new OnmsHTTPOptions(timeout, server);
```

Comment:
Declaration of `opts` should move to below where the header is set since it isn't needed until then.